src/components/calcite-value-list-item/calcite-value-list-item.tsx

```diff
@@ -19,32 +23,37 @@ export class CalciteValueListItem {
   /**
    * Compact reduces the size of the item.
    */
-  @Prop({ reflect: true }) compact = false;
+  @Prop({ reflect: true }) compact? = false;
```

Comment:
nitpick: If you set the value to false, the optional flag won't do anything.

Reply:
You would need to do `@Prop({ reflect: true }) compact?: boolean;`

Reply:
Applies to other places.

Reply:
When you say it won't do anything, what do you mean. Do you mean it won't appear as optional in the docs?

Reply:
correct

Reply:
Ok. I think having the value is more valuable. So long as the required props that have no default value show up as required in the docs I think that will suffice.

Reply:
Ok. You can remove the `?` then if you set a default value. There's no reason to say its optional when its set automatically.